lib/ansible/modules/windows/win_command.py

```diff
@@ -60,17 +60,16 @@
 '''
 
 EXAMPLES = r'''
-# Example from Ansible Playbooks.
-- win_command: whoami
+- name: Save the result of 'whoami' in 'whoami_out'
+  win_command: whoami
   register: whoami_out
 
-# Run the command only if the specified file does not exist.
-- win_command: wbadmin -backupTarget:C:\backup\ creates=C:\backup\
+- name: Create a backup only if the file does not exist
+  win_command: wbadmin -backupTarget:C:\backup\
+    creates: C:\backup\
 
-# You can also use the 'args' form to provide the options. This command
-# will change the working directory to C:\somedir\\ and will only run when
-# C:\backup\ doesn't exist.
-- win_command: wbadmin -backupTarget:C:\backup\ creates=C:\backup\
+- name: Move to C:\somedir\ and create a backup only if the file does not exists
```

Comment:
exists -> exit (should be singular).  I'm not really sure this is an improvement as you lose the explanation of what 'args' can be used for.